garage/envs/box2d/box2d_env.py

```diff
@@ -351,7 +351,7 @@ def get_com_velocity(self, *com):
         return self._compute_com_pos_vel(*com)[2:]
 
     @overrides
-    def render(self, states=None, actions=None, pause=False):
+    def render(self, mode='human', states=None, actions=None, pause=False):
```

Comment:
This violates the [gym.Env interface](https://github.com/openai/gym/blob/master/gym/core.py#L11). Only the [gym.Wrapper interface](https://github.com/openai/gym/blob/master/gym/core.py#L238) (which you can subclass) supports kwargs other than mode.